specification/syntex/resource-manager/Microsoft.Syntex/preview/2022-09-15-preview/examples/DocumentProcessor_Update.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "cd0d3cc8-cae2-4559-b882-cd707170799b",
+    "resourceGroupName": "mygroup",
+    "processorName": "myprocessor",
+    "api-version": "2022-09-15-preview",
+    "body": {
+      "tags": {
+        "additionalProp1": "string4",
+        "additionalProp2": "string5",
+        "additionalProp3": "string6"
+      },
+      "properties": {
+        "spoTenantUrl": "https://test123.sharepoint.com",
+        "spoTenantId": "e9bb744b-9558-4dc6-9e50-a3297e3332fa"
+      }
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
>200 [](http://example.com/codeflow?start=5&length=3)

Will your response ever need to run for several seconds or longer? Do you potentially want to return 202 Accepted?

Reply:
No, it will not.  This was agreed upon at the ARM board review meeting.